manifests/config/global.pp

```diff
@@ -18,15 +19,13 @@
   $flattendata = $newtype.keys.reduce({}) |$memo, $key| { $memo + {$key => $newtype[$key]["value"]} }
 
   unless empty($flattendata) {
-    concat::fragment { 'rsyslog::component::global_config::newtype':
-      order   => $::rsyslog::global_config_priority,
-      target  => "${::rsyslog::confdir}/${::rsyslog::target_file}",
-      content => epp('rsyslog/global_config.epp',
-      {
-        'type'   => 'rainerscript',
-        'config' => $flattendata,
-      }),
+    rsyslog::component::global_config { 'rainerscript':
+      *      => {
```

Comment:
I'm new to this syntax (and can't find the docs for it anywhere), but I was under the impression that the `* =>` sets defaults if no other value is set. So, if I were to list target, priority, and confdir normally, would that statically set those values as they are passed to the defined type and ignore any dynamically set values in hiera?